j2735_msgs/msg/ControlValue.msg

```diff
@@ -0,0 +1,48 @@
+#
+# ControlValue.msg
+#
+# J2735 2016 message format.
+#
+# @author Misheel Bayartsengel
+# @version 0.1
+#
+# Description
+# ...
+
+# ControlValue ::= EXPLICIT CHOICE {
+# 	value INTEGER,
+# 	direction ENUMERATED {forward, reverse},
+# 	lataffinity ENUMERATED {left, right},
+# 	perm ENUMERATED {none, permitted, passing_only, emergency_only},
+# 	prkingallowd ENUMERATED {no, parallel, angled}
+# }
+
+# Value ::= INTEGER (TODO:SIZE)
```

Comment:
Lets mark the size for now as 32bit min and 32bit max. Looks like this field has different uses depending on context so its hard to say what a good size is.